app/helpers/pulmap_geoblacklight_helper.rb

```diff
@@ -1,5 +1,22 @@
 module PulmapGeoblacklightHelper
   include GeoblacklightHelper
+  # Render a label/value constraint on the screen. Can be called
+  # by plugins and such to get application-defined rendering.
+  #
+  # Can pass in nil label if desired.
+  # @see Blacklight::RenderConstraintsHelperBehavior#render_constraint_element
+  #
+  # @param [String] label to display
+  # @param [String] value to display
+  # @param [Hash] options
+  # @option options [String] :remove url to execute for a 'remove' action
+  # @option options [Array<String>] :classes an array of classes to add to container span.
+  # @return [String]
+  def render_constraint_element(label, value, options = {})
+    value = nil if params[:bbox]
```

Comment:
this causes all of the facets to discard their value when the bbox param is present